parity/cli/mod.rs

```diff
@@ -646,7 +646,7 @@ usage! {
 
 			FLAG flag_tx_queue_no_unfamiliar_locals: (bool) = false, or |c: &Config| c.mining.as_ref()?.tx_queue_no_unfamiliar_locals.clone(),
 			"--tx-queue-no-unfamiliar-locals",
-			"Transactions received via local means (RPC, WS, etc) will be treated as external if the sending account is unknown.",
+			"Transactions received via local means (HTTP JSON-RPC, WebSockets JSON-RPC, etc) will be treated as external if the sending account is unknown.",
```

Comment:
The ambiguity of "means" in english makes me stumble when I read this.
How about "Transactions received through JSON-RPC (HTTP or WS) will be treated as external if the sending  account is unknown."

Reply:
:+1: